.travis.yml

```diff
@@ -20,6 +20,7 @@ addons:
 
 before_install:
   - chmod a+x mvnw
+  - chmod a+x travis/publish.sh
```

Comment:
`travis/publish.sh` cannot fix its own permission if it is not flagged as executable!

As far as I know git carries the +x flag already, so you can also just correct the scripts locally (or give the git command for windows) and these chmod commands should not be necessary.